internal/db/schema/migrations/postgres/2/84_oidc.up.sql

```diff
@@ -186,6 +201,63 @@ create trigger
 after update on auth_oidc_method
   for each row execute procedure update_version_column();
 
+-- active_auth_oidc_method_must_be_complete() defines a function to be used in 
+-- a "before update" trigger for auth_oidc_method entries.  Its intent: prevent
+-- incomplete oidc methods from transitioning out of the "inactive" state.
+create or replace function
+  active_auth_oidc_method_must_be_complete()
+  returns trigger
+as $$
+  begin
+    if new.state != 'inactive' then
+      perform 
+      from 
+        auth_oidc_method am
+       join auth_oidc_callback_url  cb    on am.public_id = cb.oidc_method_id 
+       join auth_oidc_signing_alg   alg   on am.public_id = alg.oidc_method_id
+      where
```

Comment:
Should this also check/verify there are non values in `auth_oidc_method` for `client_id`, `client_secret`, and probably a few others?

Reply:
ty!